spec/face.html

```diff
@@ -1098,6 +1098,122 @@
         </dl>
       </section>
     </section>
+    <section class='informative'>
+      <h2>
+        Examples
+      </h2>
+      <h3>
+        Start/Stop face module
+      </h3>
+      <pre class='example highlight'>
+        var previewStream;
+        var ft;
+
+        // Start stream firstly, then start face module.
+        startButton.onclick = function(e) {
```

Comment:
It would be better to define `startButton`, `stopButton` in the begin of the file.